src/vehicle.h

```diff
@@ -1100,7 +1091,6 @@ class vehicle : public JsonSerializer, public JsonDeserializer
     std::map<point, std::vector<int> > relative_parts;    // parts_at_relative(x,y) is used alot (to put it mildly)
     std::set<label> labels;            // stores labels
     std::vector<int> alternators;      // List of alternator indices
-    std::vector<int> fuel;             // List of fuel tank indices
```

Comment:
Why are you removing the index caches?
It's a good idea not to have to go through all the frames and doors every time you want to fetch some fuel.